packages/xo-server/src/api/vm.mjs

```
@@ -48,6 +48,44 @@ const extract = (obj, prop) => {
   return value
 }
 
+const startVmAndDestroyCloudConfigVdi = async (xapi, vm, vdiUuid, params) => {
+  try {
+    const start = new Date()
+    ignoreErrors.call(xapi.startVm(vm._xapiId))
```

I think the ignore error should be for the full function 

```suggestion
    await xapi.startVm(vm._xapiId)
```